src/runtime/mini_ion.h

```diff
@@ -0,0 +1,18 @@
+#ifndef HALIDE_RUNTIME_ION_H
+#define HALIDE_RUNTIME_ION_H
+
+#include "HalideRuntime.h"
+
+namespace Halide { namespace Runtime { namespace Internal { namespace Ion {
+
+// Allocate an ION buffer, and map it, returning the mapped
```

Comment:
It's Android. I'll try to find a place to comment, and learn what it is myself. The actual android code itself isn't very informative: https://android.googlesource.com/platform/system/core/+/master/libion/include/ion/ion.h